Oversized serving input

For `data.json` file with 7.1 mb contents, manager fails to serve model.
Stacktrace:
```
[ERROR] i.h.s.m.c.ModelServiceController i.h.s.c.ServingDataDirectives$$anonfun$completeExecutionResult$2.apply.25 Serving failed
akka.http.scaladsl.model.EntityStreamSizeException: EntityStreamSizeException: actual entity size (Some(38515045)) exceeded content length limit (8388608 bytes)! You can configure this by setting `akka.http.[server|client].parsing.max-content-length` or calling `HttpEntity.withSizeLimit` before materializing the dataBytes stream.
	at akka.http.scaladsl.model.HttpEntity$Limitable$$anon$1.preStart(HttpEntity.scala:609) ~[akka-http-core_2.11-10.0.9.jar:?]
	at akka.stream.impl.fusing.GraphInterpreter.init(GraphInterpreter.scala:290) ~[akka-stream_2.11-2.5.3.jar:?]
	at akka.stream.impl.fusing.GraphInterpreterShell.init(ActorGraphInterpreter.scala:540) ~[akka-stream_2.11-2.5.3.jar:?]
	at akka.stream.impl.fusing.ActorGraphInterpreter.tryInit(ActorGraphInterpreter.scala:659) ~[akka-stream_2.11-2.5.3.jar:?]
	at akka.stream.impl.fusing.ActorGraphInterpreter.preStart(ActorGraphInterpreter.scala:707) ~[akka-stream_2.11-2.5.3.jar:?]
	at akka.actor.Actor$class.aroundPreStart(Actor.scala:521) ~[akka-actor_2.11-2.5.3.jar:?]
	at akka.stream.impl.fusing.ActorGraphInterpreter.aroundPreStart(ActorGraphInterpreter.scala:650) ~[akka-stream_2.11-2.5.3.jar:?]
	at akka.actor.ActorCell.create(ActorCell.scala:591) ~[akka-actor_2.11-2.5.3.jar:?]
	at akka.actor.ActorCell.invokeAll$1(ActorCell.scala:462) ~[akka-actor_2.11-2.5.3.jar:?]
	at akka.actor.ActorCell.systemInvoke(ActorCell.scala:484) ~[akka-actor_2.11-2.5.3.jar:?]
	at akka.dispatch.Mailbox.processAllSystemMessages(Mailbox.scala:282) ~[akka-actor_2.11-2.5.3.jar:?]
	at akka.dispatch.Mailbox.run(Mailbox.scala:223) ~[akka-actor_2.11-2.5.3.jar:?]
	at akka.dispatch.Mailbox.exec(Mailbox.scala:234) ~[akka-actor_2.11-2.5.3.jar:?]
	at akka.dispatch.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260) [akka-actor_2.11-2.5.3.jar:?]
	at akka.dispatch.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339) [akka-actor_2.11-2.5.3.jar:?]
	at akka.dispatch.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979) [akka-actor_2.11-2.5.3.jar:?]
	at akka.dispatch.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107) [akka-actor_2.11-2.5.3.jar:?]
```
Note that log says that manager got 38515045 bytes, while data.json size is 7132595 bytes.

*Update*
This issue shows up when model runtime (in this case VGG 300 TF model) returns huge response.

## Fix
Set `akka.http.[server|client].parsing.max-content-length` parameter to ~100mb.

--- a/manager/src/main/resources/application.conf
+++ b/manager/src/main/resources/application.conf
@@ -3,6 +3,10 @@ akka {
   loglevel = "INFO"
   logging-filter = "akka.event.slf4j.Slf4jLoggingFilter"
   http.server.request-timeout = 60s
+
+  // FIX FOR https://github.com/Hydrospheredata/hydro-serving/issues/44
+  http.server.parsing.max-content-length = 200000000
+  http.client.parsing.max-content-length = 200000000
 }
 
 sidecar {

